docs/modules/ROOT/pages/ios_settings.adoc

```diff
@@ -0,0 +1,97 @@
+= Settings
+:toc: right
+:toclevels: 1
+:keywords: settings, passcode lock, biometric lock, theme, logging, ownCloud, iOS, iPhone, iPad
+:description: This guide steps you through how to configure ownCloud's iOS App for iPhone and iPad. It covers security, theme, logging, and media upload settings.
+:heic-image-url: https://en.wikipedia.org/wiki/High_Efficiency_Image_File_Format
+
+To manage the settings in ownCloud's iOS App for iPhone and iPad, click btn:[Settings] in the bottom right-hand corner of the Accounts List view.
+
+image:user-accounts-list-annotated-with-callout.png[Accessing settings in ownCloud's iOS App for iPhone and iPad.]
+
+== Security
+
+=== Passcode 
+
+To protect access to the iOS app with a Passcode, enable btn:[Passcode Lock] in the Setting's Security section.
+You will then be prompted to enter, and repeat, a 4-digit Passcode.
+
+image:settings-security-passcode-enabled.png[Enable a Passcode lock in ownCloud's iOS App for iPhone and iPad.]
+
+=== Lock Delay
+
+When a Passcode is enabled, the app will be locked every time you change to another application.
+However, under menu:Settings[Security > Lock application], you can choose to only lock the application after 1, 5, or 30 minutes, instead of "_always_", which is the default.
+
+image:settings-lock-application-duration.png[Set the application lock duration in ownCloud's iOS App for iPhone and iPad.]
+
```

Comment:
Screenshot outdated. The option `Always` was replaced by `Immediately`.

Reply:
Won't fix for now – screenshots need to be updated anyway